requirements/requirements.in

```diff
@@ -1,4 +1,4 @@
-ansible-runner==1.3.4
+ansible-runner==1.4.0
```

Comment:
👍

Reply:
Good call, had not even thought about this (guess I thought that it would be bumped outside of the PR, but makes sense to do it here instead)